services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -621,6 +622,12 @@ public void handleDeletedInstances(Set<Instance> deletedInstances) {
     }
   }
 
+  public void start() {
```

Comment:
Does `start` sounds a bit too general in this case? or add a paragraph to explain the expectation of this function.

Would it be a necessary to call HelixLiveInstanceMonitor.clear() during the graceful shutdown of a controller or it has done already, but I missed?

Reply:
Good catch, I will add a clear during shutdown. start is generic name, but we can later add other init method in future if need comes.